packages/amplify_datastore_plugin_interface/lib/amplify_datastore_plugin_interface.dart

```diff
@@ -47,13 +47,13 @@ abstract class DataStorePluginInterface extends AmplifyPluginInterface {
   late final ModelProviderInterface modelProvider;
```

Comment:
Hmm what is the reasoning for keeping this as `late final`? I think it should also just be nullable.

Reply:
It should always be initialized and have a value right?